### DISCUSSION TITLE
Ayuda con Css no puedo mandar a la derecha mi icono :(

### DISCUSSION THREAD

**Discussion by FernandoBarajasValdivia** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#461] [Q&A]
Como puedo mandar mi logo a la derecha? añado capturas
// vista
![image](https://user-images.githubusercontent.com/99165285/159107667-a98b768e-b07a-4902-a499-28b3d9f1af08.png)

//html
![image](https://user-images.githubusercontent.com/99165285/159107682-86ffea79-7d48-49e1-9979-0a44b0180c38.png)
//css

![image](https://user-images.githubusercontent.com/99165285/159107698-bf476133-7ae5-460c-9de4-f1f8745f53c6.png)

![image](https://user-images.githubusercontent.com/99165285/159107686-be7dc340-ad79-44b1-a266-d81b0a5928f5.png)

**Comment by Ingrzs** [comment]
dale 
postion: relative
postion: right  o tanteale con px

o bien
ponle margin-left : 50px; ahi tu sabras cuan distancia toma tu icono

**Reply by FernandoBarajasValdivia** [reply]
Gracias, ya con esto pude investigar y lo logré, en serio te agradezco :)
![image](https://user-images.githubusercontent.com/99165285/159111374-a9edba9b-569c-41ba-9857-a74d5678bb9f.png)

**Reply by Ingrzs** [reply]
Me da gusto saber que lo lograste :D , como dice el Mc a Darle
